executor/runtime/docker/docker.go

```diff
@@ -518,13 +500,10 @@ func (r *DockerRuntime) setupLogs(c *runtimeTypes.Container, containerCfg *conta
 	c.Env["TITUS_REDIRECT_STDOUT"] = "/logs/stdout"
 	c.Env["TITUS_UNIX_CB_PATH"] = filepath.Join("/titus-executor-sockets/", socketFileName)
 	/* Require us to send a message to tini in order to let it know we're ready for it to start the container */
-	c.Env["TITUS_CONFIRM"] = "true"
+	c.Env["TITUS_CONFIRM"] = trueString
```

Comment:
unrelated note here, @andrew-leung mentioned some of these `titus-executor -> tini` ENV vars were leaking into user entrypoints earlier this week. This may be a good opportunity to review them and `unsetenv` what's missing on `tini`.